Fix XML Syntax and other minor changes

JapNolt opened the pull request:
# Title

Fix XML Syntax and other minor changes

## Summary

Most of the changes are to fix XML formatting issues or to make the XML more readable. There are a few changes to attribute names which I note below.

## Details
[Fix attribute name in Example](https://github.com/dotnet/docs/commit/0fcad6fc02b54b5c7bf9a28ed68d0ba270263219)
[Fix attribute name in Syntax section](https://github.com/dotnet/docs/commit/86e478bfbc02da167ba8598d99aca298b0e77951)
[Change the "name" attribute to "address". According to VS Intellisense](https://github.com/dotnet/docs/commit/f42a2a5fb59827a80e47e211ef75d35ae03f6778)
[Change attribute from "name" to "type", per VS Intellisense.](https://github.com/dotnet/docs/commit/47a35bd722ebf60b953438bf18f3383616131921)
[Change attribute from "name" to "prefix", per VS intellisense](https://github.com/dotnet/docs/commit/e80904ae8d57f7d1fabd418a3ca190683dfe3686)
[Fix element name in Syntax section](https://github.com/dotnet/docs/commit/9de83243a241dde4c22ef1ded6971da6958a038f)

JapNolt committed:
Update socket-element-network-settings.md

Fix XML Syntax

JapNolt committed:
Update add-element-for-webrequestmodules-network-settings.md

Prettify XML Syntax section

JapNolt committed:
Update remove-element-for-webrequestmodules-network-settings.md

Prettify XML in Syntax and Example sections

JapNolt committed:
Update idn-element-uri-settings.md

Fix XML Syntax

JapNolt committed:
Update add-element-for-schemesettings-uri-settings.md

Prettify XML Syntax

JapNolt committed:
Update remove-element-for-schemesettings-uri-settings.md

Fix XML Syntax

JapNolt committed:
Update add-element-for-authenticationmodules-network-settings.md

Fix XML Syntax

JapNolt committed:
Update remove-element-for-authenticationmodules-network-settings.md

Prettify XML in Syntax and Example sections

JapNolt committed:
Update add-element-for-bypasslist-network-settings.md

Prettify XML in Syntax section

JapNolt committed:
Update add-element-for-connectionmanagement-network-settings.md

Prettify and fix XML in Syntax and Example sections

JapNolt committed:
Update defaultftpcachepolicy-element-network-settings.md

Fix attribute name in Example

JapNolt committed:
Update defaulthttpcachepolicy-element-network-settings.md

Fix XML in Syntax and Example sections

JapNolt committed:
Update httplistener-element-network-settings.md

Prettify XML in Syntax

JapNolt committed:
Update httpwebrequest-element-network-settings.md

Prettify XML

JapNolt committed:
Update iriparsing-element-uri-settings.md

Fix XML syntax

JapNolt committed:
Update module-element-network-settings.md

Prettify XML

JapNolt committed:
Update network-element-network-settings.md

Prettify XML

JapNolt committed:
Update proxy-element-network-settings.md

Prettify XML

JapNolt committed:
Update remove-element-for-bypasslist-network-settings.md

Prettify XML

JapNolt committed:
Update remove-element-for-connectionmanagement-network-settings.md

Fix attribute name in Syntax section and prettify the other XML

JapNolt committed:
Update remove-element-for-bypasslist-network-settings.md

Change the "name" attribute to "address". According to VS Intellisense, this attribute should be address instead of name.

JapNolt committed:
Update remove-element-for-authenticationmodules-network-settings.md

Change attribute from "name" to "type", per VS Intellisense.

JapNolt committed:
Update remove-element-for-webrequestmodules-network-settings.md

Change attribute from "name" to "prefix", per VS intellisense

JapNolt committed:
Update schemesettings-element-uri-settings.md

Prettify XML

JapNolt committed:
Update servicepointmanager-element-network-settings.md

Prettify XML

JapNolt committed:
Update settings-element-network-settings.md

Prettify XML

JapNolt committed:
Update specifiedpickupdirectory-element-network-settings.md

Prettify XML

JapNolt committed:
Update webrequestmodules-element-network-settings.md

Prettify XML

JapNolt committed:
Update system-net-element-network-settings.md

Prettify XML and link to webRequestModules

JapNolt committed:
Update webproxyscript-element-network-settings.md

Prettify XML

JapNolt committed:
Update authenticationmodules-element-network-settings.md

Prettify XML

JapNolt committed:
Update bypasslist-element-network-settings.md

Prettify XML

JapNolt committed:
Update connectionmanagement-element-network-settings.md

Prettify XML

JapNolt committed:
Update iriparsing-element-uri-settings.md

Fix element name in Syntax section

rpetrusha reviewed (changes requested):
Thank you so much, @JapNolt, for improving the formatting of these topics, as well as for identifying and fix badly formed XML tags and correcting syntax errors. In reviewing your PR, I found a few other errors. Would you mind fixing them as well? Or, if you prefer, we can fix them for you.

JapNolt committed:
Update add-element-for-authenticationmodules-network-settings.md

Change type attribute in Syntax area; Change type attribute description in Attributes area; Change remarks section to more closely match how the type is defined in the attribute

JapNolt reviewed (commented):
@rpetrusha Is this what you're looking for? I'm not an expert at DocFX formatting. Note that I also made 2 changes in the Remarks section.

JapNolt committed:
Update module-element-network-settings.md

Changes to Syntax, Attributes, Remarks, and Example sections.

JapNolt committed:
Update add-element-for-webrequestmodules-network-settings.md

Changed the Syntax, Attributes, and Remarks sections.

JapNolt committed:
Update module-element-network-settings.md

Add missing paren

JapNolt committed:
Update add-element-for-authenticationmodules-network-settings.md

Add missing closing paren

rpetrusha commented:
This is what I was looking for, @JapNolt. Thank you so much. You've done a really outstanding job! We'll merge your PR now. Changes should appear live on docs.microsoft.com in the next day or two.
